test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -492,11 +492,10 @@ hw.module @issue595_variant2_checkRedunctionAnd(%arr: !hw.array<128xi1>) {
 hw.module @if_multi_line_expr1(%clock: i1, %reset: i1, %really_long_port: i11) {
   %tmp6 = sv.reg  : !hw.inout<i25>
 
-  // CHECK:      if (reset)
+  // CHECK: if (reset)
   // CHECK-NEXT:   tmp6 <= 25'h0;
-  // CHECK-NEXT: else begin
-  // CHECK-NEXT:   automatic logic [24:0] _tmp = {{..}}14{really_long_port[10]}}, really_long_port};
-  // CHECK-NEXT:   tmp6 <= _tmp & 25'h3039;
+  // CHECK-NEXT: else
+  // CHECK-NEXT:   tmp6 <= {{..}}14{really_long_port[10]}}, really_long_port} & 25'h3039;
```

Comment:
It seems like this should have been split with a newline? Is there any problem here?